lib/runPostBuildProcess.js

```diff
@@ -17,7 +18,12 @@ module.exports = function runPostBuildProcess(anyBranchesUpdated, config) {
       ({ proc, args }) => {
         return new Promise((resolve, reject) => {
           const start = Date.now();
-          const child = spawn(proc, args);
+          const child = spawn(proc, args.map((arg) => {
+            if (arg === '${SUBDOMAINS}') {
```

Comment:
I'm curious what the config would look like in this case. Do you have an example of what that might look like?